synapseformation/client.py

```diff
@@ -39,6 +39,13 @@
 #     return config
 
 
+def _add_acl(syn, entity, acl_config):
```

Comment:
I'm not sure it would make sense to hint types with this function, but it'd be nice to keep it consistent.  It also feels to me that this function belongs in the `create.py` file, and not here.